report/main.qmd

```diff
@@ -0,0 +1,103 @@
+
+1.  Side-by-side boxplots of zero-calorie or sugary drink sales by variables such as day of the week or site can help identify if there are differences in sales between different levels of the categorical variables.
+
+2.  Plots of the proportion of sugary and zero-calorie sales coming from zero-calore drinks over counts, such as Figure 1, can be used to observe the pattern over time. These plots can also be stratified by other variables, such as intervention to identify differences between groups.
+
+3.  A scatterplot of zero-calorie vs sugary drink sales be used to investigate what relationship they have and potentially identify if they are highly correlated.
+
+Additionally, summary statistics such as means can be calculated for different variables, such as average zero-calorie sales for each day of the week to give a numeric summary.
+
+```{r,echo=FALSE,message=FALSE,warning=FALSE}
+#| label: fig-myFirstFigure
+#| fig-cap: Proportion of total sugary and zero-calorie drink sales coming from zero-calorie drinks over time
+#| 
+library(here)
+library(tidyverse)
+# Read in data
+sugary_bev <- read.csv(file = here("./rawdata/june1data.csv"),
+colClasses = c("DofW" = "factor", "Site" = "factor", "Intervention" = "factor"))
+# Plot percentage of sugary or zero sugar sales accounted for by zero sugar drinks over the counts by site
+sugary_bev %>%
+mutate(percent_zero = ZeroCal/(ZeroCal+Sugary)) %>%
+ggplot(aes(x=Count,y=percent_zero,col=Site))+
+geom_point()+
+geom_smooth(se=F)+
+scale_fill_continuous(guide = guide_legend()) +
+theme(legend.position="bottom")+
+labs(x = "Day of the Experiment",
+     y = "Proportion of Zero-Calorie \nand Sugary Drink Sales")
+```
+
+# Formal Analysis
+
+In this study, an effective intervention will increase the proportion of daily sales that come from zero-calorie beverages among sales that are either zero-calorie beverages or sugary beverages. This proportion increases when there is relatively more sales of zero-calorie beverages than of sugary beverages. This avoids the need to perform two separate analyses (one for zero-calorie and another one for sugary beverages) and any variation due to sales of other unrelated products.
+
+To answer the statistical questions, we recommend using a generalized linear mixed effects model (GLMM) with a Poisson distribution. This type of model uses count data and allows for different effects across sites. Another advantage of this model is that it can handle the longitudinal nature of the data. The response variable is the counts of zero-calorie beverage sales. Including the log-transformed total sales of zero-calorie or sugary beverages as an offset, this models the proportion of zero-calorie beverage sales as defined previously. The main independent variables of interest are intervention indicators, treating the baseline periods as the reference. Other covariates in the model include the site, the day of the week, and the study day.
```

Comment:
GEE might also be good to consider here. It can be thought of as an extension of GLMM to longitudinal data since it can handle some missing data and has no distributional assumptions. Also it can address serial correlation.